forge-gui/res/cardsfolder/upcoming/haldir_lorien_lieutenant.txt

```diff
@@ -0,0 +1,11 @@
+Name:Haldir, Lórien Lieutenant
+ManaCost:X G
+Types:Legendary Creature Elf Soldier
+PT:0/0
+K:Vigilance
+K:etbCounter:P1P1:X
+SVar:X:Count$xPaid
+A:AB$ PumpAll | Cost$ 5 G | ValidCards$ Creature.Elf+StrictlyOther+YouCtrl | NumAtt$ Y | NumDef$ Y | KW$ Vigilance | SpellDescription$ Until end of turn, other Elves you control gain vigilance and get +1/+1 for each +1/+1 counter on NICKNAME.
+SVar:Y:Count$CardCounters.P1P1
+DeckHas:Ability$Counters
+Oracle:Haldir, Lórien Lieutenant enters the battlefield with X +1/+1 counters on it.\nVigilance\n{5}{G}: Until end of turn, other Elves you control gain vigilance and get +1/+1 for each +1/+1 counter on Haldir.
```

Comment:
Lórien = Lorien

Make sure to replace all instances of accents like this in the oracle please